include/benchmark/benchmark.h

```diff
@@ -662,6 +664,7 @@ typedef void(Function)(State&);
 // chained into one expression.
 class Benchmark {
  public:
+  typedef Benchmark* (*Creator_t)();
```

Comment:
we generally don't do _ in type names: https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
I will change the code.